pkg/ccl/crosscluster/logical/logical_replication_job.go

```diff
@@ -251,51 +427,17 @@ func (r *logicalReplicationResumer) ingest(
 		execCfg.DistSQLSrv.Stopper,
 		physicalPlan,
 		execCfg.InternalDB,
-		jobID)
+		p.jobID)
```

Comment:
I think we'll want to move this StorePlanDiagram out of this function so that we only store the initial plan diagram and not every one that we re-generate.